samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_client.c

```diff
@@ -10,7 +10,9 @@
 #include <stdlib.h>
 #include <zephyr/shell/shell.h>
 #include "desh_print.h"
+
 #include "nrf_modem_dect_phy.h"
```

Comment:
Should it be?
```suggestion
#include <nrf_modem_dect_phy.h>
```

Reply:
Yeah, I think that it could be like that. But on the other hand, it is like this in other files in desh.

Reply:
Changed all like that